rpcs3/Emu/Cell/lv2/sys_fs.cpp

```diff
@@ -2116,34 +2116,9 @@ error_code sys_fs_fcntl(ppu_thread& ppu, u32 fd, u32 op, vm::ptr<void> _arg, u32
 			break;
 		}
 
-		const cfg::device_info device = g_cfg_vfs.get_device(g_cfg_vfs.dev_usb, vpath);
+		cfg::device_info device = g_cfg_vfs.get_device(g_cfg_vfs.dev_usb, vpath);
+		std::tie(arg->vendorID, arg->productID) = device.get_hex_ids();
 
-		if (device.path.empty() || device.vid.empty() || device.pid.empty())
-		{
-			arg->out_code = CELL_ENOTSUP;
-			break;
-		}
-
-		u16 vid{};
-		{
-			auto [ptr, err] = std::from_chars(device.vid.data(), device.vid.data() + device.vid.size(), vid, 16);
```

Comment:
I think this file now has an unused `#include <charconv>` that can be removed

Reply:
Removed.